cmd/tx.go

```diff
@@ -1037,3 +1038,39 @@ func ensureKeysExist(chains map[string]*relayer.Chain) error {
 
 	return nil
 }
+
+// MsgRegisterCounterpartyPayee registers the counterparty_payee
+func registerCounterPartyCmd(a *appState) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "register-counterparty chain_name channel_id port_id relay_addr counterparty_payee",
+		Aliases: []string{"reg-cpt"},
+		Short:   "register the counterparty relayer address",
```

Comment:
Can we make this slightly more verbose? Something like `register the counterparty relayer address for ics-29 fee middleware`